.github/workflows/build-test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Install Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 18
```

Comment:
I thought node 18 wasn't available in GHA yet?

Reply:
it actually is available

Reply:
Definitely available, we've been using it for a while.